regolith/builders/grantreportbuilder.py

```diff
@@ -45,19 +45,20 @@ def latex(self):
         """Render latex template"""
         rc = self.rc
         # Get Dates
-        start_date = rc.start_date.split("-")
-        end_date = rc.end_date.split("-")
-        begin_year, begin_month, begin_day = int(start_date[0]), int(start_date[1]), int(start_date[2])
-        end_year, end_month, end_day = int(end_date[0]), int(end_date[1]), int(end_date[2])
-        end_date = datetime.datetime(end_year, end_month, end_day)
-        begin_date = datetime.datetime(begin_year, begin_month, begin_day)
+        start_date = datetime.strptime(rc.start_date, "%Y-%m-%d")
```

Comment:
we usually use a date parser from dateutils.   pls could you look in one of the other helpers for the syntax?